tsconfig.strict.json

```diff
@@ -28,6 +28,7 @@
     "lib/util/date.ts",
     "lib/util/exec/common.ts",
     "lib/util/exec/types.ts",
+    "lib/util/fs",
```

Comment:
Don't we need wildcard here?

Reply:
Do you approve or not? :)

Reply:
https://www.typescriptlang.org/tsconfig#include